documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -121,6 +180,22 @@ AKS support and limitations:
 
 See also the [Azure Kubernetes Service sample]({{<relref "/samples/azure-kubernetes-service/_index.md">}}).
 
+##### Oracle WebLogic Server on the Azure Kubernetes Service Marketplace (WLS on AKS Marketplace)
+
```

Comment:
See previous comment regarding: A more accurate rendering is...

Reply:
fixed. please resolve if it looks good to you now. note that I also made a similar change to the sample cross-reference note.

Reply:
Resolved.